gnowledge/gstudio
# UI Enhancements (Registration, password-reset & Login/Logout templates)

**What to test?**
```
1) In any of the below case, keep in mind you shouldn't be getting Group's menu-bar (GApp bar along with search functionality).

2) Check login and logout templates (added images and messages)

3) Try resetting password (with valid & invalid I/Ps)

4) Register an user and watch out for templates after completion of registration and activation of that user account using link within the email.

```

**Functionalities implemented:**

**1) Enabled visibility of templates for successful completion of registration and activation of account respectively**

  - File modified:

    1) urls/__init__.py

**2) Group's menu-bar (GApp bar along with search functionality) is now made hidden for account's specific urls**

  - Via ```check_accounts_url()``` which checks whether the given path is of accounts related or not

  - Files modified:

    1) templatetags/ndf_tags.py
    2) templates/ndf/base.html

**3) Images added to be used in registration, login/logout & password-reset templates.**

  - Files created:

    1) gnowsys_ndf/ndf/static/ndf/images/login.png
    2) gnowsys_ndf/ndf/static/ndf/images/come_back_soon.png
    3) gnowsys_ndf/ndf/static/ndf/images/forgot_password.png
    4) gnowsys_ndf/ndf/static/ndf/images/password_recovery.png
    5) static/ndf/images/change_password.png

**4) Template Enhancements: 'trans' block-tag and foundation's grid retouched; new images included.**

  - Files modified:

    1) templates/registration/activation_complete.html
    2) templates/registration/activation_email.txt
    3) templates/registration/login.html
    4) templates/registration/logout.html
    5) templates/registration/password_reset_complete.html
    6) templates/registration/password_reset_confirm.html
    7) templates/registration/password_reset_done.html
    8) templates/registration/password_reset_email.html
    9) templates/registration/password_reset_error_form.html
    10) templates/registration/password_reset_form.html
    11) templates/registration/registration_complete.html
    12) templates/registration/registration_form.html
